datahub-web-react/src/app/entity/user/UserGroups.tsx

```diff
@@ -53,19 +85,35 @@ export default function UserGroups({ urn, initialRelationships, pageSize }: Prop
     const userGroups = relationships?.relationships?.map((rel) => rel.entity as CorpGroup) || [];
 
     return (
-        <GroupsView direction="vertical" size="middle">
-            <Typography.Title level={3}>Group Membership</Typography.Title>
-            <Row justify="center">
-                <GroupList
-                    dataSource={userGroups}
-                    split={false}
-                    renderItem={(item, _) => (
-                        <List.Item>
-                            {entityRegistry.renderPreview(EntityType.CorpGroup, PreviewType.PREVIEW, item)}
-                        </List.Item>
-                    )}
-                    bordered
-                />
+        <GroupsViewWrapper>
+            <Row justify="space-between">
+                {userGroups &&
+                    userGroups.map((item) => {
+                        return (
+                            <GroupItemColumn xl={8} lg={8} md={12} sm={12} xs={24} key={item.urn}>
+                                <Link to={entityRegistry.getEntityUrl(EntityType.CorpGroup, item.urn)}>
+                                    <GroupItem>
+                                        <Row className="title-row">
+                                            <GroupTitle>{item.info?.displayName || 'NA'}</GroupTitle>
```

Comment:
if displayName isnt present, lets defer to `item.name`

Reply:
<img width="850" alt="Screen Shot 2022-02-15 at 9 19 18 AM" src="https://user-images.githubusercontent.com/29713365/153989927-9f50c8d4-0e21-4dc1-b906-b899d0ebd42e.png">

With ref to above image; 
We are using the 1. 'name' field only but when we create a new group, the name is not proper format. So @jjoyce0510 ask me to use the 'displayName' instead of 'name'
So we have Extended the EnitityRelationship type by adding info as 'CorpGroupProperties' type for info key. 

I have added this change in separate types file path-> _datahub/datahub-web-react/src/app/entity/user/type.ts_

let me know if we need to change it back to 'name' only

Reply:
Yeah in new groups, the name can be a UUID. I think it's fine to still fallback to the name field, though it will be a bit jarring if that actually happens in practice

Reply:
As discussed, info.displayName || name is added.